internal/controller/argo_app.go

```diff
@@ -34,7 +34,8 @@ func (r *PaasNSReconciler) EnsureArgoApp(
 		return nil
 	}
 
-	logger := getLogger(ctx, paasns, "ArgoApp", appName)
+	ctx = setLogComponent(ctx, "ArgoApp")
```

Comment:
It this printed in log lines as well? If so, than this should be de-capitalised as well?

Reply:
I've gone ahead and changed all component names to lowercase